test/bin/build_images.sh

```diff
@@ -362,6 +362,15 @@ do_group() {
 
     if ${BUILD_INSTALLER} && ! ${COMPOSER_DRY_RUN}; then
         for image_installer in "${groupdir}"/*.image-installer; do
+            # If a template arg was given, only build the image installer for the
```

Comment:
I'm curious - can you give me a command that has wrong behavior without the fix? I think I noticed some unnecessary installer builds, but I can't recall the exact scenario.

Reply:
test/bin/build_images.sh -E -i -f -t test/image-blueprints/layer1-base/group1/rhel93.toml

Reply:
Argh, I get it now - this one doesn't show up until you fix the primary one...